core/src/main/scala/org/apache/spark/sql/rapids/tool/qualification/QualificationAppInfo.scala

```diff
@@ -198,24 +198,67 @@ class QualificationAppInfo(
     }
   }
 
+  private def checkStageIdInExec(prev: Option[ExecInfo],
+      execInfo: ExecInfo, next: Option[ExecInfo]): (Seq[(Int, ExecInfo)], Option[ExecInfo]) = {
```

Comment:
function isn't descriptive enough for me to know what these returns are, can you add javadoc explaining.

Reply:
added documentation. PTAL.